spryker/nginx/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/continuouspipe/symfony-php7.1-nginx:latest
+
+MAINTAINER Hardik Gajjar <hardik.gajjar@inviqa.com>
+
+RUN apt-get update && \
+    apt-get install -y php7.1-pgsql php7.1-gmp postgresql-client graphviz redis-tools
```

Comment:
Don't forget to clean up the apt caches after this.

Try to combine the two RUN commands into one, a good example is: https://github.com/continuouspipe/dockerfiles/blob/master/drupal8-apache/7.0/Dockerfile#L5-L15